### ISSUE TITLE
Process doesn't close when minimizing to tray

### ISSUE THREAD

**Issue by supertrio17** [TcNobo/TcNo-Acc-Switcher#158]
I had this exact same problem before, I reinstalled my windows in the time being, and I hoped this would fix it. but it doesn't. Program just wont kill itself. When I close, it starts the tray process but it doesn't kill the main one, so I can't start it again unless I kill it myself via task manager.

**Comment by TcNobo** [comment]
I assume you're getting a message popup when double-clicking the tray icon? If so I can recreate this, and a fix should be on the way.

**Comment by TcNobo** [comment]
Try downloading the pre-release here, and let me know if things are fixed :)
https://github.com/TcNobo/TcNo-Acc-Switcher/releases/tag/2022-02-07_00

**Comment by supertrio17** [comment]
![image](https://user-images.githubusercontent.com/57882120/152861914-960ef214-bf66-408d-8db6-8ee371dd0445.png)
![image](https://user-images.githubusercontent.com/57882120/152861937-b4669c0f-94ed-4f40-9e3a-a8f3b63ca8ef.png)
Still the same. Any ideas why its not working?

**Comment by supertrio17** [comment]
Sorry, it actually does work, there is a problem when uninstalling. First one is that all files don't get deleted:
![image](https://user-images.githubusercontent.com/57882120/152863313-830bf45b-e156-4831-882d-9ed4a93c04df.png)
And the second one is that settings in roaming do not get deleted.

**Comment by supertrio17** [comment]
I like how its instantly showing now, and not taking time before starting, because the process is still alive, just hidden. Thanks :)

**Comment by TcNobo** [comment]
Changing settings needs the `Close` button at the bottom to be clicked to save changes.
While it is nice to have it start quicker, I'm also getting this issue. If I can't figure out a more reliable way to get this to work, I'll just have to resort to completely closing it and completely reopening it as it was originally :/
